Sources/Plasma/NucleusLib/pnNetBase/pnNbConst.h

```diff
@@ -49,15 +49,11 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 *
 ***/
 
-// FIXME: Remove all Windows assumptions in the net code
-#ifndef MAX_PATH
-#   define MAX_PATH 260
-#endif
-
 //============================================================================
 // Network constants
 //============================================================================
 const unsigned kMaxTcpPacketSize                = 1460;
+const unsigned kCli2File_FilenameSize           = 260;
```

Comment:
I didn't realize MAX_PATH was being used for a bunch of string outside of File Server stuff... would `kNetDefaultStringSize` be a better name?

(Would it have killed them to use Pascal-style strings in their network messages? This is so wasteful 🤦🏼)

Reply:
Probably would be a better name. It is not great that the Windows maximum path length got baked into so many things.